docs/iris/src/whatsnew/1.10.rst

```diff
@@ -2,7 +2,7 @@ What's New in Iris 1.10
 ***********************
 
 :Release: 1.10
-:Date: 2016-08-08
+:Date: 2016-09-05
```

Comment:
This is a bit picky, but the last several whatsnew pages have had release dates specified in the format `nth Month YYYY`, and it would be nice to see that here too please.